src/data/markdown/docs/02 javascript api/06 k6-execution.md

```diff
@@ -71,7 +71,7 @@ Control the test execution.
 
 | Property | Type | Description |
 |----------|------|-------------|
-| abort | function | It immediately aborts a k6 load test run with the exit code `108`. Aborting a test is fully supported in `k6 cloud`, preserving the same user experience for both local and cloud users. |
+| abort | function | It immediately aborts a k6 load test run with the exit code `108`. |
```

Comment:
```suggestion
| abort([String]) | function | It aborts a `k6` load test run with the exit code `108`, and optionally, a String can be passed for providing a more detailed error's message. `k6` executes the `teardown()` function if it has been defined before stopping the execution. |
```
WDYT?

Reply:
Suggestion:

It aborts the test run with the exit code `108`, and an optional string parameter can provide an error message. Aborting the test will not prevent the [`teardown()`]() execution.